### PR TITLE
DOCS: Update dependencies for Qt6

### PR DESCRIPTION
**Pull request by Hartmnt**

### CODE REVIEW

File: docs/dev/build-instructions/build_linux.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ sudo apt install \
   build-essential \
   cmake \
   pkg-config \
-  qt5-qmake \
-  qtbase5-dev \
-  qttools5-dev \
-  qttools5-dev-tools \
-  libqt5svg5-dev \
+  qmake6 \
```
Comment (Krzmbrzl) [inline docs/dev/build-instructions/build_linux.md:17]: We shouldn't need qmake anymore

File: docs/dev/build-instructions/build_linux.md [hunk 2]
```diff
@@ -119,7 +116,6 @@ poco-devel \
 gcc-toolset-9-gcc-c++ 
 ```
 
-You will need `cmake` **v3.15 or later**. You can install a recent one from the [linked page](https://cmake.org/download/).
```
Comment (Krzmbrzl) [inline docs/dev/build-instructions/build_linux.md:122]: This is still up-to-date, isn't it? 👀

Comment (Hartmnt) [reply docs/dev/build-instructions/build_linux.md:122]: According to pkgs.org Centos 9 Stream ships with cmake >= 3.20

Comment (Krzmbrzl) [reply docs/dev/build-instructions/build_linux.md:122]: Ah, I see :+1:

**Review by Krzmbrzl — APPROVED** [review]

**Comment by davidebeatrici** [comment]
About the Ubuntu dependencies: https://github.com/mumble-voip/mumble/blob/1a379c98fcded69330647c024ef67d237656f98f/.github/actions/install-dependencies/install_ubuntu_shared_64bit.sh#L34-L46

Maybe we should explain the two different branches in a textual way.

**Comment by Hartmnt** [comment]
I have added ``qt6-l10n-tools`` and ``libgl-dev``
However, apt automatically selects ``qt6-svg-dev`` over ``libqt6svg6-dev`` on 24.04 so this should not be a problem

**Comment by davidebeatrici** [comment]
Excellent. Also, the dependencies we are explicitly installing for 22.04 are pulled in automatically in 24.04, meaning that there are no side effects aside from the packages being marked as manually installed.

**Review by davidebeatrici — APPROVED** [review]

**Pull request merged**